README.md

```diff
@@ -1 +1,18 @@
-# Welcome to OpenBC Website
\ No newline at end of file
+# Welcome to OpenBC
+
+## Project Set Up Instructions
+
+1. Clone the repo `openbc-web` using SSH
```

Comment:
might be helpful to show the command
i.e.

`git clone git@github.com:OpenBCca/openbc-web.git`